### ISSUE TITLE
[MessageStrip]: The close button does not have an aria-label.

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#11142]
### Bug Description

[MessageStrip]: The close button does not have an aria-label.

### Affected Component

MessageStrip

### Expected Behaviour

[MessageStrip]: The close button should have arial-label and it should be same as title, otherwise screen reader may read both arial-label  and title which normally will have same meaning.

This issue was mentioned in https://github.com/SAP/ui5-webcomponents/issues/9874

### Priority

None

### UI5 Web Components Version

v2.X.X

### Browser

Chrome, Edge, Firefox, Safari

### Operating System

All

### Organization

SuccessFactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @pankajpandey01, 

The MessageStrip already has a properly defined tooltip according to the mentioned issue https://github.com/SAP/ui5-webcomponents/issues/9874. 
Why would it need a label, as having it will cause a double announcement. 

Kind regards, 
Georgi

**Comment by anusandhanam** [comment]
Hi @Valkanister,

   We have requested this change to be in alignment with the UI5 framework's message strip implementation which has aria-labelledby attribute in addition to title attribute.

![Image](https://github.com/user-attachments/assets/43f9034b-dfc4-4c9e-889c-eb7045828f46)

![Image](https://github.com/user-attachments/assets/7efe5715-3b13-46a4-a2ee-2b7e8df16a39)

Also, this is in alignment with WCAG recommendations to use aria-label/aria-labelledby for accessible names instead of titles as best practice.

![Image](https://github.com/user-attachments/assets/852e950b-d539-4654-9b09-f6d926e61d75)

Thank you

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-rl 
Please have a look at this feature request for the MessageStrip. It aims to equalize the ACC behavior between it and the standard control. 

Kind regards, 
Georgi
